atomic

```diff
@@ -119,6 +116,15 @@ if __name__ == '__main__':
     parser.add_argument('--debug', default=False, action='store_true')
     subparser = parser.add_subparsers(help=_("commands"))
 
+    # atomic cluster (if available)
+    try:
+        import commctl.cli
+        clusterp = subparser.add_parser(
```

Comment:
No problem.  For the record, though, I was following the example of `atomic host` which is also written to be optional.